charts/fluentd/values.yaml

```diff
@@ -290,100 +290,108 @@ dashboards:
 plugins: []
 # - fluent-plugin-out-http
 
+defaultConfigs:
+  prometheusConf:
+    enabled: true
+  systemdConf:
+    enabled: false
+
 ## Add fluentd config files from K8s configMaps
 ##
 configMapConfigs:
-  - fluentd-prometheus-conf
-# - fluentd-systemd-conf
+# - 01_sources.conf
```

Comment:
Dosen't make sense to need to add the values specifically on fileConfigs and here. This should be automatic!!

As request of many users, we shall comment out the definitions under (or mabe set them as a config example only?)
```yaml
fileconfigs: []
```
So, it solves this issue.